instant-distance-py/src/lib.rs

```diff
@@ -24,41 +26,84 @@ fn instant_distance_py(_: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<Search>()?;
     m.add_class::<Hnsw>()?;
     m.add_class::<HnswMap>()?;
+    m.add_class::<DistanceMetric>()?;
     Ok(())
 }
 
+#[pyclass]
+#[derive(Copy, Clone, Default)]
+enum DistanceMetric {
+    #[default]
```

Comment:
This will probably violate the MSRV. For now, I think adding a "manual" `Default` impl is probably fine?